### PR TITLE
iPhone SE Styling

### PR DESCRIPTION
**Pull request by shin-kinoshita**
## Background
This application is working for every iOS device, but useless for iPhone SE because of styling problem.
I'm a fan of SE and this app, that's why I decided to modify the style.
Hope this PR to be merged and release my modification, please!

## Implementation
- [x] modify style for iPhone SE
  I just changed iPhoneSE style only..
  But I do not have any knowledge and experiences of UI design.
  So please review the design modification.

  I will attach screenshots of each screen.
  
  image
  image
  image

## Question
- oy value of Card.tsx
  After my modification, card position while animating to show Result, became a little bit unnatural.
  Maybe, it was caused because I changed card size and paddingTop of container in Home.
  Finally I found that oy field in Card.tsx should be adjusted to make the animation natural, but I could not understand how to calculate it.
  Now the value is temporally adjusted with some try and error of watching and changing it.
  If any right calculation way exists, please tell me.

### CODE REVIEW

**Comment by maro-12** [comment]
@shin-kinoshita 
Thank you for iPhoneSE correspondence!!
But there are conflicting files... Please resolve😣

File: src/util/DisplaySize.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Dimensions, Platform } from 'react-native';
+
+const SE_WIDTH = 320;
+const SE_HEIGHT = 568;
+
+export const isIPhoneSe = () => {
```
Comment (35d) [inline src/util/DisplaySize.tsx]: [nits] typo?

I prefer SE to Se.

**Comment by 35d** [comment]
@shin-kinoshita Thank you for your pull request ✨
I made some comments. Would you be able to check them?

> oy value of Card.tsx 

Even when we made it, we adjusted the position by repeating trial and error. So, I think we have no choice but to try and error of watching and changing even if iPhone SE.

File: src/util/DisplaySize.tsx (hunk 1 above)
Comment (shin-kinoshita) [reply src/util/DisplaySize.tsx]: fixed the character

**Comment by shin-kinoshita** [comment]
@maro-12 @35d 
Thanks for some comments.
I finished the below tasks, please check them.
- [x] resolving conflict files
- [x] modifying function name, isIPhoneSe to isIPhoneSE
- [x] checking oy position (actually no need to change)

**Comment by 35d** [comment]
@shin-kinoshita 

Thank you for the modification.
When I tried moving it in my simulator of iPhoneSE, it looked pretty good.

Please correct only one point.
Please change the value of `oy` to the following.

```
const oy = isIPhoneSE()
  ? (Dimensions.get('window').height + 60 - 307) / 2
  : (Dimensions.get('window').height - 20 - 307) / 2;
```

**Comment by shin-kinoshita** [comment]
@35d 
I modified oy to the one you specified.

**Pull request merged**